### DISCUSSION TITLE
recipes/breads/best-ever-banana-bread/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#1101] [Comments]
# recipes/breads/best-ever-banana-bread/

A collection of my recipes.

https://nicholaswilde.io/recipes/breads/best-ever-banana-bread/

**Comment by nicholaswilde** [comment]
- [ ] use combination method from Claire

**Reply by nicholaswilde** [reply]
- https://youtube.com/watch?v=CilyToUevz0
